encode.ts

```diff
@@ -0,0 +1,36 @@
+function toPostgresArray(array: Array<any>): string {
+    let postgresArray = "{";
+
+    array.forEach((element, index) => {
+        if (index > 0) {
+            postgresArray += ",";
+        }
+
+        if (!element) {
+            postgresArray += "NULL";
+        } else {
+            postgresArray += encode(element);
```

Comment:
this is wrong, there elements are not escaped... check it here: https://github.com/brianc/node-postgres/blob/fcd0f02210f532bf2d823ea05105f4aaade389eb/lib/utils.js#L25